ush/gaussian_sfcanl.sh

```diff
@@ -181,6 +184,8 @@ cat <<EOF > fort.41
   igaus=${LONB_SFC},
   jgaus=${LATB_SFC},
   donst=${do_nst},
+  imp_physics=${imp_physics},
+  landsfcmdl=${landsfcmdl},
```

Comment:
```suggestion
  imp_physics=${imp_physics:-8},
  landsfcmdl=${landsfcmdl:-2},
```
This is the same as defining those local variables above on 145/146